src/main/index.js

```diff
@@ -512,9 +512,9 @@ function runApp() {
       backgroundColor: windowBackground,
       darkTheme: nativeTheme.shouldUseDarkColors,
       icon: process.env.NODE_ENV === 'development'
-        ? path.join(__dirname, '../../_icons/iconColor.png')
+        ? path.join(__dirname, '../../_icons/iconColor.svg')
         /* eslint-disable-next-line n/no-path-concat */
-        : `${__dirname}/_icons/iconColor.png`,
+        : `${__dirname}/_icons/iconColor.svg`,
```

Comment:
This file needs to stay as a png, Electron doesn't support using svgs as window icons.

Reply:
[Source here](https://github.com/electron/electron/issues/9642). Good catch!